lib/capabilities.js

```diff
@@ -36,7 +36,11 @@ var capabilities = {
     checks: [
       {
         check: function(cb) {
-          cb(false, 'Appmetrics does not support smart profiling.');
+          if (agent().internal.supports.watchdog) {
```

Comment:
does this mean that a capability check causes appmetrics to be required, with side effect of starting it? that would not be good